pincer/objects/message/user_message.py

```diff
@@ -546,11 +546,16 @@ def clean_content(self):
         """
         The message content with any special characters removed.
         """
-        remove_bold = sub(r"\*\*(.*?)\*\*", r"\1", self.content)
-        remove_italic1 = sub(r"\*(.*?)\*", r"\1", remove_bold)
-        remove_italic2 = sub(r"_(.*?)_", r"\1", remove_italic1)
-        remove_bold_italic = sub(r"\*\*\*(.*?)\*\*\*", r"\1", remove_italic2)
-        remove_underline = sub(r"\_\_(.*?)\_\_", r"\1", remove_bold_italic)
+        new_content = self.content
+        subs = [
```

Comment:
i guess can be extract out of the function to a constant

Reply:
Can you explain what you mean by this, I don't understand

Reply:
I am assuming that @Sigmanificient is saying that the regex can be a constant in the code, defined at the top outside of the functions/classes.

Reply:
yep